src/expressServer.ts

```diff
@@ -255,7 +255,7 @@ function initializeRoutes(
   app.use(apiErrorMiddleware);
 
   app.use(function onError(err: any, _req: any, res: any, _next: any) {
-    logger.error(`Fallthrough error: ${err}`);
```

Comment:
What exactly is a fallthrough error?

Reply:
If no express middleware pick up the error, it comes to this final fall through handler.